Taskfile.yml

```diff
@@ -3,15 +3,18 @@ version: '3'
 vars:
   MODULE_BASE_PATH: "github.com/jbarzegar/ron-mod-manager" 
   RON_MM_CONFIG_DIR: "{{.HOME}}/.config/ron-mod-manager"
-  LDFLAGS: "{{.MODULE_BASE_PATH}}/config.ConfPath={{.RON_MM_CONFIG_DIR}}"
+  CFGLDFLAGS: "{{.MODULE_BASE_PATH}}/config.ConfPath={{.RON_MM_CONFIG_DIR}}"
+
+  DB_CONFIG_PATH: "{{.HOME}}/.local/state/ron-mod-manager/ron-mod-manager-state.sqlite"
+  DBLDFLAGS: "{{.MODULE_BASE_PATH}}/config.DBPath={{.DB_CONFIG_PATH}}"
```

Comment:
Is there an cleaner way of dealing with LDflags? I feel like this is gonna become messy in the long run

Reply:
Could opt to instead have an ld flag like `mode=development` and then have these paths codified inside of the Go instead?